pettingzoo/classic/chess/chess.py

```diff
@@ -202,10 +205,21 @@ def action_space(self, agent):
         return self.action_spaces[agent]
 
     def observe(self, agent):
-        observation = chess_utils.get_observation(
-            self.board, self.possible_agents.index(agent)
-        )
+        agent_index = self.possible_agents.index(agent)
```

Comment:
I think you should make the variable naming here consistent with the step function, which does the following:
```
current_agent = self.agent_selection
current_index = self.agents.index(current_agent)
```
Note that self.possible_agents always has both agents, whereas self.agents can have one agent removed if they do an illegal move and terminate, which would make the next observation incorrect.
I can see an argument that agent_index is a better name than current_index, but probably best to just keep the way things were dont previously and rename your var to current_index. 

TLDR probably want to just do `current_index = self.agents.index(current_agent)` and replace `agent_index` on the next line with `current_index`.

Reply:
I replace the `agent_index` with `current_index` and also check the naming consistency.